python/lsst/ts/cRIOpy/M1M3TS/ThermalValuePageWidget.py

```diff
@@ -0,0 +1,195 @@
+# This file is part of M1M3 GUI.
+#
+# Developed for the LSST Telescope and Site Systems.
+# This product includes software developed by the LSST Project
+# (https://www.lsst.org). See the COPYRIGHT file at the top - level directory
+# of this distribution for details of code ownership.
+#
+# This program is free software : you can redistribute it and / or modify it
+# under the terms of the GNU General Public License as published by the Free
+# Software Foundation, either version 3 of the License, or (at your option) any
+# later version.
+#
+# This program is distributed in the hope that it will be useful, but WITHOUT
+# ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+# FOR A PARTICULAR PURPOSE.See the GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License along with
+# this program.If not, see <https://www.gnu.org/licenses/>.
+
+from PySide2.QtWidgets import (
+    QWidget,
+    QTableWidget,
+    QPushButton,
+    QTableWidgetItem,
+    QVBoxLayout,
+    QHBoxLayout,
+    QGridLayout,
+    QSpinBox,
+)
+from PySide2.QtCore import Slot
+from ..GUI import TopicWindow, SALCommand
+from .ThermalData import Thermals
+
+from asyncqt import asyncSlot
+
+BUTTON_FANS = 1
+BUTTON_HEATERS = 2
+
+
+class DataWidget(QTableWidget):
+    def __init__(self):
+        super().__init__(10, 10)
+        for r in range(0, 10):
+            for c in range(0, 10):
+                item = QTableWidgetItem("")
+                self.setItem(r, c, item)
+        self.empty()
+
+    def empty(self):
+        for r in range(0, 10):
+            for c in range(0, 10):
+                self.item(r, c).setText(str(r * 10 + c))
+
+
+class CommandWidget(QWidget):
+    class SetButton(QPushButton):
```

Comment:
I realize this is an internal class, but again a doc string would be helpful.

Is it necessary that this class be defined within the CommandWidget class? If not, I strongly suggest defining it at the module level because it makes the code easier to read.